src/module/item/mixins/item-activation.js

```diff
@@ -60,7 +60,6 @@ export const ItemActivationMixin = (superclass) => class extends superclass {
         updateData['system.isActive'] = active;
 
         const updatePromise = this.update(updateData);
-        const rollMode = game.settings.get("core", "rollMode");
```

Comment:
I think it might be best to keep this to allow for sending of text as private to the GM, if needed